utility/build.gradle

```diff
@@ -64,6 +64,7 @@ dependencies {
       'androidx.appcompat:appcompat:1.0.2',
       'androidx.lifecycle:lifecycle-livedata-ktx:2.2.0-alpha03',
       'androidx.work:work-runtime-ktx:2.4.0',
+      'com.github.anandwana001:kotlitex:3e8a0804041121ece4b57099ac3de386d78e0996',
```

Comment:
Using the latest commit, we might use `branch = master` in the Bazel file and `-master:SNAPSHOT` in gradle to always use the latest commit from the master branch. 
This will reduce updating the commit anytime when we change anything in the kotlitex.

Reply:
I think we actually will want to snapshot to a specific version (like we do elsewhere). Even though we control the repository, it's helpful to keep builds as deterministic as possible so there are no unexpected surprises.